core/ts/MaciState.ts

```diff
@@ -1876,6 +1880,7 @@ const genProcessVkSig = (
         BigInt(_voteOptionTreeDepth)
 }
 
+// TODO: Not used anywhere.Also one for new key generation is missing.
 const genDeactivationVkSig = (
     _messageQueueSize: number,
     _stateTreeDepth: number,
```

Comment:
`genNewKeyGenerationVkSig` is missing in `MaciState`

Reply:
Yes, i left that in the TODO comment because i was not sure do we need these as the genDeactivationVkSig is not used anywhere. Let's resolve once Aleksandar is back,

Reply:
Correct, but right now it should serve to initialize the VkRegistry contract with the key also of the `GenerateKeyFromDeactivated` circuit.